students/grzegorz_bajorski/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,9 @@
+x1 = int(input())
+y1 = int(input())
+x2 = int(input())
+y2 = int(input())
+
+z = (x1-x2) * (y1-y2)
+
+print('YES') if (abs(z) == 2) else print('NO')
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.